src/controllers/editPost.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable camelcase */
+const { join } = require('path');
+const { editPostQuery } = require('../database/queries');
+
+const getEditPost = (req, res) => {
+  res.sendFile(
+    join(__dirname, '..', '..', 'public', 'html', 'editPost.html'),
+  );
+};
+const editPost = (req, res, next) => {
+  const { title, content_post, image_url } = req.body;
+  const { id } = req.params;
+  editPostQuery(id, {
+    title,
+    content_post,
+    image_url,
+  })
+    .then((data) => {
+      res.status(201).json({
```

Comment:
It would be more appropriate to use  200 as an HTTP status code in the response.